doc/input-output-reference/src/overview/group-internal-gains-people-lights-other.tex

```diff
@@ -132,22 +132,6 @@ \subsubsection{Inputs}\label{inputs-025}
 
 Number of People * People Schedule * People Activity * Carbon Dioxide Generation Rate. The default value is 3.82E-8 m3/s-W (obtained from ASHRAE Standard 62.1-2007 value at 0.0084 cfm/met/person over the general adult population). The maximum value can be 10 times the default value.
 
-\paragraph{Field: Cold Stress Temperature Threshold}\label{cold-stress-temperature-threshold}
-
-This field is used in the calculation of hours of safety for cold events, which
-is defined as the longest duration (number of hours), starting from the
-beginning time of the risk period (e.g., the start time of a power outage), to
-not below a certain temperature threshold (specified in this field). If not
-specified, 60F (or 15.56C) will be used as the default temperature threshold.
-
-\paragraph{Field: Heat Stress Temperature Threshold}\label{heat-stress-temperature-threshold}
-
-This field is used in the calculation of hours of safety for hot events, which
-is defined as the longest duration (number of hours), starting from the
-beginning time of the risk period (e.g., the start time of a power outage), to
-not above a certain temperature threshold (specified in this field). If not
-specified, 86F (or 30C) will be used as the default temperature threshold.
```

Comment:
These were moved/removed?  I didn't expect that from the PR text.  Could just be a groggy start for me though :)

Reply:
Yes, they were moved because the order did not match the IDD before.  In the current IDD, these are placed at the end (N7, N8) of the object syntax.  I moved them to match the IDD.